config/initializers/session_limitable.rb

```diff
@@ -19,6 +19,8 @@
 
   if warden.authenticated?(scope) && options[:store] != false
     if record.unique_session_id != current_session_id
+      analytics = Analytics.new(user: record, request: warden.request, session: {}, sp: nil)
```

Comment:
We may be able to get the session at this point from `warden.raw_session` ?

(And if we really wanted, maybe the `sp` via the `warden.raw_session`'s `sp` metadata?)

Reply:
good call, added in 3207699